deepchem/feat/molecule_featurizers/mol_graph_conv_featurizer.py

```diff
@@ -0,0 +1,196 @@
+  - Hydrogen bonding: A one-hot vector of whether this atom is a hydrogen bond donor or acceptor.
+  - Aromatic: A one-hot vector of whether the atom belongs to an aromatic ring.
+  - Degree: A one-hot vector of the degree (0-5) of this atom.
+  - Number of Hydrogens: A one-hot vector of the number of hydrogens (0-4) that this atom connected.
+
+  The default edge representation are constructed by concatenating the following values,
+  and the feature length is 11.
+
+  - Bond type: A one-hot vector of the bond type, "single", "double", "triple", or "aromatic".
+  - Same ring: A one-hot vector of whether the atoms in the pair are in the same ring.
+  - Conjugated: A one-hot vector of whether this bond is conjugated or not.
+  - Stereo: A one-hot vector of the stereo configuration of a bond.
+
+  If you want to know more details about features, please check the paper [1]_ and
+  utilities in deepchem.utils.graph_conv_utils.py.
+
+  Examples
+  --------
+  >>> smiles = ["C1CCC1", "C1=CC=CN=C1"]
+  >>> featurizer = MolGraphConvFeaturizer()
+  >>> out = featurizer.featurize(smiles)
+  >>> type(out[0])
+  <class 'deepchem.feat.graph_data.GraphData'>
+  >>> out[0].num_node_features
+  38
+  >>> out[0].num_edge_features
+  11
+
+  References
+  ----------
+  .. [1] Kearnes, Steven, et al. "Molecular graph convolutions: moving beyond fingerprints."
+     Journal of computer-aided molecular design 30.8 (2016):595-608.
+
+  Notes
+  -----
+  This class requires RDKit to be installed.
+  """
+
+  def __init__(self, add_self_loop: bool = False):
+    """
+    Parameters
+    ----------
+    add_self_loop: bool, default False
+      Whether to add self-connected edges or not. If you want to use DGL,
+      you sometimes need to add explict self-connected edges.
+    """
+    self.add_self_loop = add_self_loop
+
+  def _featurize(self, mol: RDKitMol) -> GraphData:
+    """Calculate molecule graph features from RDKit mol object.
+
+    Parameters
+    ----------
+    mol: rdkit.Chem.rdchem.Mol
+      RDKit mol object.
+
+    Returns
+    -------
+    graph: GraphData
+      A molecule graph with some features.
+    """
+    try:
+      from rdkit import Chem
+      from rdkit.Chem import AllChem
+    except ModuleNotFoundError:
+      raise ValueError("This method requires RDKit to be installed.")
+
+    # construct atom and bond features
+    try:
+      mol.GetAtomWithIdx(0).GetProp('_GasteigerCharge')
+    except:
+      # If partial charges were not computed
+      AllChem.ComputeGasteigerCharges(mol)
+
+    h_bond_infos = construct_hydrogen_bonding_info(mol)
+    sssr = Chem.GetSymmSSSR(mol)
+
+    # construct atom (node) feature
+    atom_features = np.array(
+        [
+            _construct_atom_feature(atom, h_bond_infos, sssr)
+            for atom in mol.GetAtoms()
+        ],
+        dtype=np.float,
+    )
+
+    # construct edge (bond) information
+    src, dist, bond_features = [], [], []
```

Comment:
You probably want the variable to be called `dest` (short for destination?), not `dist` which sounds like it's short for "distance"?

Reply:
This is short for destination. I fixed!